mlflow/server/__init__.py

```diff
@@ -188,6 +201,8 @@ def _build_gunicorn_command(gunicorn_opts, host, port, workers, app_name):
         bind_address,
         "-w",
         str(workers),
+        "-k",
+        "uvicorn.workers.UvicornWorker",
```

Comment:
for pypy, need "uvicorn.workers.UvicornH11Worker"
https://www.uvicorn.org/deployment/#gunicorn